generators/client/templates/react/src/main/webapp/app/shared/reducers/authentication.ts.ejs

```diff
@@ -176,33 +181,15 @@ export const login = (username, password, rememberMe = false) => async (dispatch
       Storage.session.set(AUTH_TOKEN_KEY, jwt);
     }
   }
- <%_ if (enableTranslation) { _%>
   await dispatch(getSession());
-
-  const account = getState().authentication.account;
-  if (account && account.langKey) {
-    await dispatch(setLocale(account.langKey));
-  }
-  <%_ } else { _%>
-  dispatch(getSession());
-  <%_ } _%>
 };
 <%_ } else if (authenticationType === 'uaa') { _%>
 export const login = (username, password, rememberMe = false) => async (dispatch, getState) => {
   const result = await dispatch({
     type: ACTION_TYPES.LOGIN,
     payload: axios.post('auth/login', { username, password })
   });
- <%_ if (enableTranslation) { _%>
   await dispatch(getSession());
```

Comment:
Same no await

Reply:
changed the logic here, use `await` only when the `tranlation is enabled`

Reply:
use await only when the tranlation is enabled which match the test cases too